docs/data/base/components/trap-focus/ContainedToggleTrappedFocus.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
+import TrapFocus from '@mui/base/TrapFocus';
+
+export default function BasicTrapFocus() {
```

Comment:
The name doesn't match

```suggestion
export default function ContainedToggleTrappedFocus() {
```